test/test_torch.py

```diff
@@ -5292,8 +5290,9 @@ def test_batch_norm_cpu_inference(self):
             input3 = input1.contiguous(memory_format=torch.channels_last)
             for name, param in m.named_parameters():
                 if param.requires_grad:
-                    if param.data.dim() == 4:
-                        param.data = param.data.contiguous(memory_format=torch.channels_last)
+                    if param.dim() == 4:
+                        with torch.no_grad():
+                            param = param.contiguous(memory_format=torch.channels_last)
```

Comment:
this doesn't look right to me -- this is just creating a new parameter, it doesn't actually change the model parameter to be channels_last, does it?

Reply:
Let me double check these and send a PR.

Reply:
This does create a new parameter upon further inspection. I got caught as above - the .data is needed to refer to the underlying tensor of the `parameter`. It should probably be `param.data = param.contiguous(memory_format=torch.channels_last)` without the context manager.

Reply:
Well, given that `affine=False`, there is no learnable parameters. So maybe the right fix is just to remove this whole for loop :D 
@gchanan removing the for loop is good? Or should we change affine to True?

Reply:
Actually @gchanan already commented about this in the original PR: https://github.com/pytorch/pytorch/pull/28982/files#r342780225 

I'll add a check for affine=True and send a PR.